docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,410 @@
+{
```

Comment:
<p>Please use</p><p> </p><pre class="ql-syntax" spellcheck="false">plt.style.use('https://raw.githubusercontent.com/TDAmeritrade/stumpy/main/docs/stumpy.mplstyle')
</pre><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
will be considered in the next commit.